src/pages/plugins/index.js

```diff
@@ -95,21 +101,28 @@ function PluginFooter({ keywords = [] }) {
 }
 
 function PluginComponent({ plugin }) {
+  const deprecatedCard = plugin.deprecatedMessage ? true : false;
+  const deprecatedMessage = deprecatedCard && plugin.deprecatedMessage;
+  const cardClasses = classnames('card', { [styles.pluginDeprecatedCard]: plugin.deprecatedMessage ? true : false });
+  const cardHeader = classnames( 'card__header', deprecatedCard ? styles.deprecatedCardHeader : styles.pluginCardHeader );
+  const cardBody = classnames( deprecatedCard ? styles.pluginCardFlagMessage : 'card__body' );
+
   return (
     <div className="row padding-vert--md">
       <div className="col col--10 col--offset-1">
-        <div key={plugin.key} className="card">
-          <div className={classnames('card__header', styles.pluginCardHeader)}>
+        <div key={plugin.key} className={cardClasses}>
+          <div className={cardHeader}>
+            {deprecatedCard && <span className="badge badge--primary">Deprecated</span>}
             <h2><a className={styles.primaryUrl} href={plugin.primaryUrl}>{plugin.name}</a></h2>
-            <span className="badge badge--primary">{plugin.version}</span>
+            {deprecatedCard || <span className="badge badge--primary">{plugin.version}</span>}
           </div>
-          <div className="card__body">
-            {plugin.description}
+          <div className={cardBody}>
+            {deprecatedCard ? ( <h4 className={classnames(styles.deprecatedMessage)}>{deprecatedMessage}</h4> ) : plugin.description}
```

Comment:
Don't use an <h4> for this, we should just use normal text element with the correct styles. Also don't use `classnames()` for one class.

Reply:
Did change <h4> to <span> tag, and understood of classnames().